doc/developer/design/20231110_aws_connections.md

```diff
@@ -107,16 +112,54 @@ When creating the connection, users should specify `WITH (VALIDATE = false)` in
 This is because we won't be able to validate the connection unless the trust policy has
 been added on the user's end.
 
+#### `mz_aws_connections` table
+
+We'll add a new `mz_aws_connections` table with the following definition:
+
+```sql
+CREATE TABLE mz_internal.mz_aws_connections (
+    -- The ID of the connection in `mz_connections`.
+    id text NOT NULL,
+    -- The value of the `ENDPOINT` option, if specified. `NULL` otherwise.
+    endpoint text,
+    -- The value of the `REGION` option, if specified. `NULL` otherwise.
+    region text,
+    -- The value of the `AWS ACCESS KEY ID` option, if specified. `NULL` otherwise.
+    access_key_id text,
```

Comment:
We allow the access_key to be either a string or a secret. So we shouldn't display it here?

Reply:
Oh, good point! We should have two different columns then: `aws_access_key_id text` and `aws_access_key_id_secret_id text`. Which one is filled in, if one is filled in at all, depends on whether you provided the access key as a string or a secret.

Reply:
Updated as described.